src/main/java/sirius/web/security/GenericUserManager.java

```diff
@@ -157,6 +171,47 @@ private UserInfo loginViaSSOToken(WebContext ctx) {
         return null;
     }
 
+    private UserInfo loginViaCookie(WebContext ctx) {
+        String user = ctx.getCookieValue("user");
+        String token = ctx.getCookieValue("token");
+
+        if (Strings.isFilled(user) && Strings.isFilled(token)) {
+            ctx.hidePost();
+
+            UserInfo result = findUserByName(ctx, user);
+            if (result != null) {
+                // The cookie token is TIMESTAMP:MD5
+                Tuple<String, String> challengeResponse = Strings.split(token, ":");
+                // Verify age...
+                if (checkTokenTtl(Value.of(challengeResponse.getFirst()).asLong(0), loginCookieTtl * 24 * 60 * 60)) {
```

Comment:
umrechnung mit TimeUnit....oooder loginCookieTTL als "Duration" definieren und in der config mit "90 days" angeben <- schöner